src/modules/formBuilder/formBuilderUtil.ts

```diff
@@ -0,0 +1,24 @@
+import { ItemMap } from '../form/types';
```

Comment:
Ah, this is a better place for this, since it's a util not a component. We should move the contents of [components/formBuilderUtil.ts](https://github.com/greenriver/hmis-frontend/blob/release-119/src/modules/formBuilder/components/formBuilderUtil.ts) into here

Reply:
Ah okay, I moved those over